src/JWTUtils/Services/HMACSHA256SignatureService.cpp

```diff
@@ -21,7 +21,9 @@ namespace systelab { namespace jwt {
 		unsigned int len = 32;
 		unsigned char hash[32];
 
-#if (OPENSSL_VERSION_NUMBER >= 0x10101000)
+#if (OPENSSL_VERSION_NUMBER >= 0x30000000L)
```

Comment:
Do we want to keep compatibility with older versions of openssl that are no longer under support?